core/src/main/scala/org/scalawag/jibe/mandate/command/package.scala

```diff
@@ -0,0 +1,37 @@
+package org.scalawag.jibe.mandate
+
+import java.io.File
+
+import org.scalawag.jibe.mandate.command.CommandArgument._
+
+package object command {
+  implicit val StringToValue = new ToValue[String, StringValue] {
```

Comment:
These implicits are required by the generated macros to turn other objects into Values (that's my generic decomposition target).  If you want to add a type that can be decomposed to a Command, this is where you need to add a converter for it.